Application/EdFi.Ods.AdminApi/Program.cs

```diff
@@ -23,22 +24,13 @@
     app.UseForwardedHeaders();
 }
 
-// Configure the HTTP request pipeline.
-if (app.Environment.IsDevelopment())
-{
-    app.UseDeveloperExceptionPage();
-}
-else
-{
-    app.UseExceptionHandler("/error");
-}
```

Comment:
This kind of exception handling is appropriate for an HTML-based web page, but not appropriate for an API.